docs/TranslatableInputs.md

```diff
@@ -134,6 +135,20 @@ const Selector = () => {
 </TranslatableInputs>
 ```
 
+## `StackProps`
+
+Use the `StackProps` prop to pass props to the rendered MUI [Stack](https://mui.com/material-ui/react-stack/) component.
+
```

Comment:
You should  explain what you're doing in the next example, and provide a screenshot of the result.